Civi/Api4/Generic/EntityInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+namespace Civi\Api4\Generic;
+
+/**
+ * This is currently a null interface - it allows picking
+ * up AbstractEntities via the ClassScanner
+ */
+interface EntityInterface {
```

Comment:
Oh, yeah, moving toward a straight-up `interface` would be a silver-lining. (*Tho obviously not the main goal.*)